Dockerfile

```diff
@@ -11,6 +11,7 @@ COPY --from=terraform /bin/terraform /bin/terraform
 COPY . .
 
 RUN yum -y install git openssh jq which \
+  && yum -y update openssl-libs glib2 vim-minimal vim-data curl \
```

Comment:
@thpang I did consider that approach briefly but the potential issue with that approach is that you could end up pulling in newer versions that contain Critical or High CVEs that the current versions don't have. That can result in needing to address an even greater number of CVEs than your existing image already has. The choice to only update the specific RPM bundles identified as having high CVEs was intentional. Also, by keeping them together in one update record it will allow us to more easily remove just those updates if desired when moving to a newer base image.